<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowp</class>
 <widget class="QMainWindow" name="MainWindowp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1087</width>
    <height>681</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gérer les produits</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/resources/img/image2.jpg);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="Ajoutera">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>1081</width>
      <height>621</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Ajouter</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>841</width>
        <height>481</height>
       </rect>
      </property>
      <property name="title">
       <string>GroupBox</string>
      </property>
      <widget class="QPushButton" name="pushButton_3">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>350</y>
         <width>111</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>Quitter</string>
       </property>
      </widget>
      <widget class="QPushButton" name="submit">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>350</y>
         <width>101</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>Submit</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_mat">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>30</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_nom">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>80</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Matricule</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>90</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Nom </string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_desc">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>140</y>
         <width>311</width>
         <height>121</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>140</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Description</string>
       </property>
      </widget>
      <widget class="QPushButton" name="mailling">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>270</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Verifier Mail</string>
       </property>
      </widget>
      <widget class="QLabel" name="image_3">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>290</y>
         <width>411</width>
         <height>161</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Modifier</string>
     </attribute>
     <widget class="QLineEdit" name="matm">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>70</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="nomm">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>110</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="descm">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>140</y>
        <width>171</width>
        <height>101</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_12">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>430</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Quitter</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_11">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>430</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>491</width>
        <height>441</height>
       </rect>
      </property>
      <property name="title">
       <string>GroupBox</string>
      </property>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>150</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Description</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>70</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Nom </string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>40</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>mat</string>
       </property>
      </widget>
     </widget>
     <zorder>groupBox_2</zorder>
     <zorder>matm</zorder>
     <zorder>nomm</zorder>
     <zorder>descm</zorder>
     <zorder>pushButton_12</zorder>
     <zorder>pushButton_11</zorder>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Afficher</string>
     </attribute>
     <widget class="QPushButton" name="trier">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>307</y>
        <width>93</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Tri Avancé</string>
      </property>
     </widget>
     <widget class="QPushButton" name="rechercher">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>210</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Rechercher</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="matricule_rechercher">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>180</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Ajouter_26">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>460</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Ajouter_27">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>460</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Ajouter_28">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>460</y>
        <width>51</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Next</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Ajouter_29">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>460</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>3</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pdf">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>370</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>PDF</string>
      </property>
     </widget>
     <widget class="QTableView" name="tabp">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>551</width>
        <height>391</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_5">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>315</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>matricule</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>nom</string>
       </property>
      </item>
     </widget>
     <widget class="QTableView" name="tab_rechercher">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>240</y>
        <width>241</width>
        <height>171</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>180</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Matricule:</string>
      </property>
     </widget>
     <widget class="QLabel" name="image_2">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>10</y>
        <width>411</width>
        <height>161</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <zorder>tabp</zorder>
     <zorder>trier</zorder>
     <zorder>rechercher</zorder>
     <zorder>matricule_rechercher</zorder>
     <zorder>Ajouter_26</zorder>
     <zorder>Ajouter_27</zorder>
     <zorder>Ajouter_28</zorder>
     <zorder>Ajouter_29</zorder>
     <zorder>pdf</zorder>
     <zorder>comboBox_5</zorder>
     <zorder>tab_rechercher</zorder>
     <zorder>label_5</zorder>
     <zorder>image_2</zorder>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Mailing</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>20</y>
        <width>871</width>
        <height>481</height>
       </rect>
      </property>
      <property name="title">
       <string>GroupBox</string>
      </property>
      <widget class="QPushButton" name="pushButton_4">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>420</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Quitter</string>
       </property>
      </widget>
      <widget class="QPushButton" name="submitmail">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>420</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Envoyer</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="maila">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>100</y>
         <width>471</width>
         <height>271</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="nommail">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>70</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>70</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Nom </string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>110</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Mail</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_id">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>30</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>ID</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Affichage mail</string>
     </attribute>
     <widget class="QTableView" name="tabm">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>581</width>
        <height>401</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pdf_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>390</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>PDF</string>
      </property>
     </widget>
     <widget class="QPushButton" name="trier_mail">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>340</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Tri Avancé</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_6">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>340</y>
        <width>91</width>
        <height>26</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>matricule</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>nom</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>200</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Nom</string>
      </property>
     </widget>
     <widget class="QTableView" name="tab_rechm">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>260</y>
        <width>241</width>
        <height>171</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="recherchermail">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>230</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Rechercher</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="nomaff">
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>200</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_9">
     <attribute name="title">
      <string>Suppression</string>
     </attribute>
     <widget class="QPushButton" name="Ajouter_34">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>480</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Ajouter_35">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>480</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Ajouter_36">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>480</y>
        <width>51</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Next</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Ajouter_37">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>480</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>3</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="mats">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>50</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_36">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>50</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Matricule</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>19</x>
        <y>29</y>
        <width>361</width>
        <height>331</height>
       </rect>
      </property>
      <property name="title">
       <string>GroupBox</string>
      </property>
      <widget class="QPushButton" name="pushButton_13">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>270</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Suprimer</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>40</y>
        <width>461</width>
        <height>321</height>
       </rect>
      </property>
      <property name="title">
       <string>GroupBox</string>
      </property>
      <widget class="QPushButton" name="image">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>210</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>exportation image</string>
       </property>
      </widget>
      <widget class="QTableView" name="label_pic">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>20</y>
         <width>241</width>
         <height>171</height>
        </rect>
       </property>
      </widget>
     </widget>
     <zorder>groupBox</zorder>
     <zorder>Ajouter_34</zorder>
     <zorder>Ajouter_35</zorder>
     <zorder>Ajouter_36</zorder>
     <zorder>Ajouter_37</zorder>
     <zorder>mats</zorder>
     <zorder>label_36</zorder>
     <zorder>groupBox_5</zorder>
    </widget>
    <widget class="QWidget" name="tab_6">
     <attribute name="title">
      <string>suppmail</string>
     </attribute>
     <widget class="QGroupBox" name="tabms">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>691</width>
        <height>431</height>
       </rect>
      </property>
      <property name="title">
       <string>GroupBox</string>
      </property>
      <widget class="QPushButton" name="pushButton_14">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>380</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Suprimer</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_37">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>60</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>id</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="idsupp">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>60</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_7">
     <attribute name="title">
      <string>Map</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1087</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
